### ISSUE TITLE
ReferenceError: function is not defined for function using axios  

### ISSUE THREAD

**Issue by ademsas** [DevExpress/testcafe#4802]
### Are you requesting a feature or reporting a bug?
Reporting a bug. I'm new to JS/React so go easy on me. 

### What is your Test Scenario?
1. I would like use axios to make a POST call
2. I would like to log that response
3. Then retrieve a key value pair from the response

### What is the Current behavior?
1. It is returning RefrenceError saying that the function being called `createApiOrder` is not defined. 

```js
➜  testcafe git:(td-cs-tools-api-requests) ✗ testcafe chrome fixtures/aadmin/cs_tools/api_requests.js -t 'Filter by Order public id (C35961)' -e --inspect-brk
Debugger listening on ws://127.0.0.1:9229/0c5b43bb-49fc-442c-87ff-9fe4598ac4a8
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
Using locally installed version of TestCafe.
 Running tests in:
 - Chrome 80.0.3987 / Mac OS X 10.14.6

 ApiRequests
 ✖ Filter by Order public id (C35961)

   1) ReferenceError: createApiOrder is not defined

      Browser: Chrome 80.0.3987 / Mac OS X 10.14.6

         45 |      console.log(error);
         46 |    })
         47 |}
         48 |
         49 |export async function getOrderPublicId() {
       > 50 |  const orderApiResponse = await createApiOrder()
         51 |    .then(res => console.log(res))
         52 |    .catch (err => console.error(err))
         53 |  debugger
         54 |  console.log(orderApiResponse)
         55 |

         at <anonymous> (/Users/tishdemsas/Projects/testcafe/helpers/api_requests.js:50:28)
         at getOrderPublicId (/Users/tishdemsas/Projects/testcafe/helpers/api_requests.js:62:55)
         at <anonymous> (/Users/tishdemsas/Projects/testcafe/fixtures/aadmin/cs_tools/api_requests.js:129:28)



 1/1 failed (47s)
```
### What is the Expected behavior?
- Successfully cal createApiOrder from getOrderPublicId function 
- Log the response
- Retrieve the id from the response (This bit of code should be trivial but haven't completed it yet. I need above 2 results first)

#### Provide the test code and the tested page URL (if applicable)

```js
export default async function createApiOrder(t) {
  
  const orderData = {
    "order_number": "TestSimpleOrder35",
	  "origin_address": {
		  "name": "Test Name 1",
		  "company": "TestCompany1",
		  "zip": "90027"
	  },
	  "destination_address": {
		  "name": "Test name 2",
		  "company": "Test company 2",
		  "zip": "75229"
	  }
  };

  
  return await axios( {
    method: 'post',
    url: baseUrl,
    data: orderData,
    headers: {
      'Content-type': 'application/json',
      'X-Api-Key': '2eee1f8b7bf7bcb1fdf0ca7b7e8e3c1c'
    }
  })
    .then(response => {
      // console.log(response.data)
      return response.data
    })
    .catch(error => {
      console.log(error);
    })
}
 
export async function getOrderPublicId() {
    const orderApiResponse = await createApiOrder()
    .then(res => console.log(res))
    .catch (err => console.error(err))
    console.log(orderApiResponse)
  }
```
### Your Environment details:
 
* testcafe version:                  1.3.3
* node.js version:                   v10.14.2
* command-line arguments:  testcafe chrome fixtures/aadmin/cs_tools/api_requests.js -t 'Filter by Order public id (C35961)' -e --inspect-brk
* browser name and version: Chrome 80.0.3987
* platform and version:          Mac OS X 10.14.6

**Comment by AlexSkorkin** [comment]
According to the [`export default async` makes it impossible to reference fn by name in file (T7295)](https://github.com/babel/babel/issues/3786) thread, this is a known issue in babel, which is fixed in 7.0.0.

See also: [exporting a default async function is not defined in current file](https://stackoverflow.com/questions/41203540/exporting-a-default-async-function-is-not-defined-in-current-file).

Either update babel, or do not mark createApiOrder as 'export default' async.

**Comment by AndreyBelym** [comment]
I think Alex meant to say that this issue will be fixed after we upgrade Babel to v7.0.0 in the context of https://github.com/DevExpress/testcafe/issues/3425.
